app/components/markdown/markdown_image/index.tsx

```diff
@@ -240,7 +240,7 @@ const MarkdownImage = ({
                         <ProgressiveImage
                             forwardRef={ref}
                             id={fileInfo.id!}
-                            defaultSource={{uri: fileInfo.uri!}}
+                            imageUri={fileInfo.uri}
```

Comment:
changing it like this will cause the image to be adjacent to a `Thumbnail` component that is not available unless progressive Image is modified not to mount `Thumbnail` when `thumbnailUri` is not set. but the real question is, do we care that the `Thumbnail` component is mounted?

Reply:
If there is no thumbnailUri, it will use the default thumbnail. So it will always show something. I don't remember what this is used for, so it would be great to have a refresher on where this thumbnail is supposed to show.

Reply:
yeah, we don't have thumbnails for inline images only for file attachments